compat/fsmonitor/fsmonitor-fs-listen-win32.c

```diff
@@ -525,6 +588,23 @@ int fsmonitor_fs_listen__ctor(struct fsmonitor_daemon_state *state)
 		data->nr_listener_handles++;
 	}
 
+	/*
+	 * While our CWD is the root of the worktree we can easily get the
+	 * spelling of the shortname of ".git" (if the volume has shortnames
+	 * enabled).  For most worktrees this value will be "GIT~1", but we
+	 * don't want to assume that.  If shortnames are disabled, we'll get
+	 * back the original ".git" value.  Silently ignore errors.
+	 */
+	strbuf_init(&data->dot_git_shortname, 0);
+	GetShortPathNameA(".git", shortname, sizeof(shortname));
```

Comment:
I think the immediate reaction on the Git mailing list will be "why don't you handle Unicode variations on macOS?"...

Reply:
I'm going to have to do some homework here and look at the is_hfs* routines that we have.  I'll log a new issue to track it.  Thanks!